src/styles.css

```diff
@@ -40,4 +40,7 @@
 }
 .follow-them-grid{
```

Comment:
if there's nothing in this class, it can be removed from CSS file

Reply:
Done